sql/parse/describe_test.go

```diff
@@ -0,0 +1,74 @@
+package parse
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	errors "gopkg.in/src-d/go-errors.v1"
+	"gopkg.in/src-d/go-mysql-server.v0/sql"
+	"gopkg.in/src-d/go-mysql-server.v0/sql/expression"
+	"gopkg.in/src-d/go-mysql-server.v0/sql/plan"
+)
+
+func TestParseDescribeQuery(t *testing.T) {
+	testCases := []struct {
+		query  string
+		result sql.Node
+		err    *errors.Kind
+	}{
+		{
+			"DESCRIBE TABLE foo",
```

Comment:
Describe table should work too. Is it still being parsed somewhere?

Reply:
Yep, just not here